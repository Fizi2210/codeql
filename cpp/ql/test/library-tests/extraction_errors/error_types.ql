import cpp

from Expr e
where e.getType() instanceof ErroneousType
and e.fromSource()
select e
