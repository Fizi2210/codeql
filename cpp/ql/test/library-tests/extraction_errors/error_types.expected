| test.cpp:6:13:6:31 | <error expr> |
